rasa/engine/training/hooks.py

```diff
@@ -25,47 +27,44 @@ def __init__(self, cache: TrainingCache, model_storage: ModelStorage):
     def on_before_node(
         self,
         node_name: Text,
+        graph_component_class: Type,
         config: Dict[Text, Any],
         received_inputs: Dict[Text, Any],
     ) -> Dict:
         """Calculates the run fingerprint for use in `on_after_node`."""
-        fingerprint_data = {
-            "node_name": node_name,
-            "config": config,
-            "inputs": received_inputs,
-        }
-        fingerprint_key = rasa.shared.utils.io.deep_container_fingerprint(
-            fingerprint_data
-        )
+        logger.debug(f"TrainingHook.on_before_node running for node {node_name}.")
```

Comment:
I'd  move this log to the `graph node` so we don't have do add it to every new hook class again
```suggestion
        logger.debug(f"Hook '{self.class.__name__}.on_before_node' running for node {node_name}.")
```

Reply:
what do you mean?

Reply:
we said we wouldn't have any implementation in interfaces

Reply:
ah, but the `GraphNode` isn't a interface, right? I'd move it there. Agree to not do any logging in the `Hook` interface itself

Reply:
oooh gotcha 👍  makes sense